librz/analysis/dwarf_process.c

```diff
@@ -262,10 +255,8 @@ static const char *dwarf_reg_name(RZ_NONNULL char *arch, ut64 reg_num, int bits)
 		} else {
 			return map_dwarf_reg_to_x86_reg(reg_num);
 		}
-	} else if (!strcmp(arch, "ppc")) {
-		if (bits == 64) {
-			return map_dwarf_reg_to_ppc64_reg(reg_num);
-		}
+	} else if (!strcmp(arch, "ppc") && bits == 64) {
+		return map_dwarf_reg_to_ppc64_reg(reg_num);
```

Comment:
We should support PPC32 too, not in this PR though cc @wargio @Rot127

Reply:
yes